keyboards/kv/revT/keymaps/default/keymap.c

```diff
@@ -0,0 +1,77 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/* Keymap (Base Layer) Default Layer
+   *.----------------------------------------------------------------------------------------------------------------.
+   *|                           |Esc|   | F1| F2| F3| F4| | F5| F6| F7| F8| | F9|F10|F12|F12|     |Pscr| Slck | Pause|
+   *| Nlck|  / |  * |  -  |     | ~ |  1|  2|  3|  4|  5|  6|  7|  8|  9|  0| - | = |Backsp |     | Inc| Home |  PgUp|
+   *|----------------------------------------------------------------------------------------------------------------|
+   *| NP7 | NP8| NP9|     |     |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P| [ | ] |  \  |     | Del|  End |  PgDw|
+   *|---------------|  +  |------------------------------------------------------------------------------------------|
+   *| NP4 | NP5| NP6|     |     |CAPS   |  A|  S|  D|  F|  G|  H|  J|  K|  L| ; | ' |Return |                        |
+   *|---------------|-----|------------------------------------------------------------------------------------------|
+   *| NP1 | NP2| NP3|     |     |Shift   |  Z|  X|  C|  V|  B|  N|  M| , | . | / |   Shift  |          |  Up  |      |
+   *|---------------|Enter|------------------------------------------------------------------------------------------|
+   *|    0     | Del|     |     |Ctrl|Win |Alt |        Space          |Alt | Win |FN |Ctrl |     |Left| Down | Right|
+   *`----------------------------------------------------------------------------------------------------------------'
+   */
+[0] = LAYOUT_default(
+                                        KC_ESC,           KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,    \
+  KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,   KC_GRV,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,   KC_INS,  KC_HOME, KC_PGUP,    \
+  KC_KP_7, KC_KP_8, KC_KP_9, KC_PPLS,   KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,   KC_DEL,  KC_END,  KC_PGDN,    \
+  KC_KP_4, KC_KP_5, KC_KP_6,            KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,                                  \
+  KC_KP_1, KC_KP_2, KC_KP_3, KC_PENT,   KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,                   KC_RSFT,            KC_UP,               \
+  KC_KP_0,          KC_PDOT,            KC_LCTL, KC_LGUI, KC_LALT,                KC_SPC,                                   KC_RALT, KC_RGUI, MO(1),   KC_RCTL,   KC_LEFT, KC_DOWN, KC_RIGHT),  \
```

Comment:
The trailing backslashes here and in the next LAYOUT block can be safely removed, if you like, but it's ok to leave them in if you prefer.

Reply:
I prefer to leave them